cvos/c_iraf_priv.c

```diff
@@ -4,8 +4,10 @@
 */
 const char CVOS_VERSION[] = {"CVOS Version 3.5 (10-April-2001)"};
 
+# include <stdio.h>
```

Comment:
I don't understand...

`/opt/homebrew/Cellar/gcc@12/12.4.0/lib/gcc/12/gcc/aarch64-apple-darwin23/12/include-fixed/stdio.h:413:18: note: 'fpos_t' is defined in header '<stdio.h>'; did you forget to '#include <stdio.h>'?`
`make[2]: *** [cvos/CMakeFiles/cvos.dir/c_iraf_priv.c.o] Error 1`
`make[1]: *** [cvos/CMakeFiles/cvos.dir/all] Error 2`